crates/minecraft_packets/src/play/set_player_position_and_rotation.rs

```diff
@@ -0,0 +1,10 @@
+use minecraft_protocol::prelude::*;
+
+#[derive(PacketIn)]
+pub struct SetPlayerPositionAndRotationPacket {
+    pub x: f64,
+    pub feet_y: f64,
+    pub z: f64,
+    pub yaw: f32,
+    pub pitch: f32,
```

Comment:
A field is missing in this packet:

> Flags 	Byte	Bit field: 0x01: on ground, 0x02: pushing against wall.

Before 1.21.4 it is a boolean

> On Ground 	Boolean	True if the client is on the ground, false otherwise.